x-pack/metricbeat/module/azure/client.go

```diff
@@ -81,7 +81,7 @@ func (client *Client) InitResources(fn mapMetric, report mb.ReporterV2) error {
 	// users could add or remove resources while metricbeat is running so we could encounter the situation where resources are unavailable, we log and create an event if this is the case (see above)
 	// but we return an error when absolutely no resources are found
 	if len(metrics) == 0 {
-		return errors.New("no resources were found based on all the configurations options entered")
+		client.Log.Error("no resources were found based on all the configurations options entered")
```

Comment:
I wonder if this should be a debug message?

Reply:
I have no preference here, since this is not necessary an exception it could be a debug message instead